api/sheets/get_name.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Get the name of a Data Commons node given its dcid.
+nav_order: 4
+parent: SHEETS
+grand_parent: API
+---
+
+# Get the name of a node.
+
+## =c(dcids)
```

Comment:
what is =c ?

Reply:
That should be DCGETNAME, I was trying to copy/paste :)